test/e2e/manage_existing_resources_test.go

```diff
@@ -0,0 +1,100 @@
+package e2e
+
```

Comment:
Maybe we can rename this test file name to `porting-workloads-test` or something else that is related to feature #1039 , so other related e2e can be placed here.

Reply:
Sure, `porting-workloads-test` LGTM